### ISSUE TITLE
Destructive autocorrect and false positive with SpaceInHtmlTag

### ISSUE THREAD

**Issue by PedroAugustoRamalhoDuarte** [Shopify/erb-lint#272]
With this following code:

```html
<div class="modal fade" id=<%= modal_id %> tabindex="-1" role="dialog" aria-labelledby="modalTitle" aria-hidden="true">
</div>
```
when we run with `erblint --enable-linters space_in_html_tag -a`

```html
<div class="modal fade" id=<%= modal_id %>>
```

I know that should double quotes around rails code, but rails runs this code fine and i think we can improve the linter to handle this situation.
